### PR TITLE
Homework Submitted - Sandeep

### PR DESCRIPTION
**Pull request by sandeeppal1991**

### CODE REVIEW

File: mimsmind0.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+def the_game(random_number,no_of_digits):
+    print("Let's play the mimsmind0 game.")
+    #starting of the game
+    user_input = input("Guess a {} digit number : ".format(no_of_digits))
+    no_of_tries = 0
+    while True:
+        try:
+            user_input = int(user_input)
+        except
```
Comment (danielsgriffin) [inline mimsmind0.py]: Did this run for you?

`except:`

File: mimsmind0.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            #exception raised if user input is not an integer
+            user_input = input("Invalid Input. Try again : ")
+            continue
+        no_of_tries += 1
+        #case when the guessed number is the same as the guessed number
+        if(user_input == random_number):
+            print("Congratulations ! You guessed the number in {} tries".format(no_of_tries))
```
Comment (danielsgriffin) [inline mimsmind0.py]: "You guessed the number in 1 tries"

(1) "1 try."
(2) add period

File: mimsmind1.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
+"""usage: python mimsmind1.py [length]
+Once again, the program generates a random number with number of digits equal to
```
Comment (danielsgriffin) [inline mimsmind1.py]: One run through I got this:
```
Invalid input. Try again : 123
1 bulls(s) , 0 cow(s). Try again : 145
0 bulls(s) , 1 cow(s). Try again : 256
0 bulls(s) , 1 cow(s). Try again : 527
1 bulls(s) , 0 cow(s). Try again : 428
2 bulls(s) , 0 cow(s). Try again : 424
2 bulls(s) , 0 cow(s). Try again : 422
2 bulls(s) , 0 cow(s). Try again : 423
2 bulls(s) , 0 cow(s). Try again : 429
2 bulls(s) , 0 cow(s). Try again : 420
3 bulls(s) , 0 cow(s). Try again : 420
3 bulls(s) , 0 cow(s). Try again : 44
Sorry, you couldn't guess the number in 11 tries. The number was 420
```
?

File: mimsmind1.py [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+At each round, if the user guess is incorrect and maxrounds is not yet reached,
+the program should increment the counter for round and issue a new prompt for
+user input. Otherwise, the program should print a congratulatory or an
+apologetic message with the number of guesses made, and terminate the game."""
+
+###############################################################################
+#import section
+from random import randint
+import sys
+import math
```
Comment (danielsgriffin) [inline mimsmind1.py]: you didn't use this?

File: mimsmind0.py (hunk 1 above)
Comment (sandeeppal1991) [inline mimsmind0.py]: I ran yhis and it was working fine. Must have missed the colon while adding comments. Will edit and commit.

File: mimsmind1.py (hunk 3 above)
Comment (danielsgriffin) [inline mimsmind1.py]: Following the second example:
```
Let's play the mimsmind game. You have 11 tries to guess the number
Guess a 3 digit number : 012
1 bulls(s) , 0 cow(s). Try again : 100
1 bulls(s) , 0 cow(s). Try again : 700
1 bulls(s) , 1 cow(s). Try again : 007
3 bulls(s) , -1 cow(s). Try again : 007
3 bulls(s) , -1 cow(s). Try again : 007
```

**Comment by danielsgriffin** [comment]
Note that for mimsmind it doesn't run w/out that added colon

Seems like you are really close. But the game isn't ending right. 
I'd avoid using dictionaries - needless complexity.

**Pull request closed without merging**